src/locales/index.ts

```diff
@@ -1,7 +1,7 @@
 import { en } from './en'
 
 // If you wish to define additional locales, add them here separated by `|`
-export type DefinedLocales = 'en'
+export type DefinedLocales = 'en|de'
```

Comment:
This should be two strings separated by `|`
```suggestion
export type DefinedLocales = 'en'|'de'
```